README.md

```diff
@@ -18,14 +18,7 @@
 
 ## Overview
 
-Feast (**Fea**ture **St**ore) is an open source feature store for machine learning. Feast is the fastest path to manage existing infrastructure to productionize analytic data for model training and online inference.
-
-
-Feast allows ML platform teams to:
-
-* **Make features consistently available for training and serving** by managing an _offline store_ (to process historical data for scale-out batch scoring or model training), a low-latency _online store_ (to power real-time prediction)_,_ and a battle-tested _feature server_ (for serving pre-computed features online).&#x20;
-* **Avoid data leakage** by generating point-in-time correct feature sets so data scientists can focus on feature engineering rather than debugging error-prone dataset joining logic. This ensure that future feature values do not leak to models during training.
-* **Decouple ML from data infrastructure** by providing a single data access layer that abstracts feature storage from feature retrieval, ensuring models remain portable as you move from training models to serving models, from batch models to realtime models, and from one data infra system to another.
+Feast is an open source feature store for machine learning. Feast is the fastest path to productionizing analytic data for model training and online inference.
```

Comment:
btw @kevjumba why were these changes made? seems like they don't match `docs/README.md`

Reply:
Think this was my bad. Seems like I had missed the jinja template update. Added https://github.com/feast-dev/feast/pull/2994/files to fix

Reply:
stamped #2994